src/app/component/project-option/renderer/renderer-configuration.component.html

```diff
@@ -9,4 +9,7 @@
   <div>
     <mat-slide-toggle [(ngModel)]="configuration.showGrid" name="show boards" (change)="callback()">Show boards </mat-slide-toggle>
   </div>
+  <div>
```

Comment:
While we do spoke about putting this setting in the Advanced tab, I have to say it looks weird to be in the `renderer` settings, **since it do not impact rendering**. It may fit better in the export tab, what are your thoughts on this ?

Reply:
Yeah, I have to agree it did feel odd as I was writing it. Yeah, I'll put the checkbox for that inside the export tab 😄. Thanks

Reply:
You mean slider ? ;D

Reply:
Yes, sorry my bad :). I moved this over, I followed the functionality of how the renderer component works with configuration. This might be overkill for what we need right now but makes it open to change if needed at a later point. Again let me know if you are happy with the new approach.

Reply:
I don't find it overkill, IMO it makes sense regarding the others components. While of course we have only one field to modify and play with, it is ready for accepting new features now :)